### PR TITLE
separate cmd flag of controller and node

### PR DESCRIPTION
**Pull request by zwwhdls**

### CODE REVIEW

File: cmd/node.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+		for {
+			http.ListenAndServe(fmt.Sprintf("localhost:%d", port), nil)
+			port++
+		}
+	}()
+
+	// enable pod manager in csi node
+	if !process && podManager && config.KubeletPort != "" && config.HostIp != "" {
+		if err := controller.StartReconciler(); err != nil {
+			klog.V(5).Infof("Could not Start Reconciler: %v", err)
```
Comment (Hexilee) [inline cmd/node.go]: I think `klog.Fatalln` is better here.

**Review by Hexilee — APPROVED** [review]
LGTM

File: cmd/node.go (hunk 1 above)
Comment (zwwhdls) [reply cmd/node.go]: done

**Comment by zwwhdls** [comment]
break change, closing

**Pull request closed without merging**